spec/models/pet_spec.rb

```diff
@@ -54,31 +55,109 @@
     expect(last_version.cats.first.name).to(eq("Sylvester"))
   end
 
-  it "can reify a subclassed item when item_type refers to the base_class" do
-    cat = Cat.create(name: "Garfield")        # Index 0
-    cat.update_attributes(name: "Sylvester")  # Index 1 - second
-    cat.update_attributes(name: "Cheshire")   # Index 2 - third
-    cat.destroy                               # Index 3 - fourth
-    # With PT <= v9.1 a subclassed version's item_type referred to the base_class, but
-    # now it refers to the class itself.  In order to simulate an entry having been made
-    # in the old way, set one of our versions to be "Animal" instead of "Cat".
-    versions = PaperTrail::Version.order(:id)
-    versions.second.update(item_type: cat.class.base_class.name)
-
-    animal = Animal.create                    # Index 4
-    animal.update(name: "Muppets Drummer")    # Index 5
-    animal.destroy                            # Index 6
-
-    versions = PaperTrail::Version.order(:id)
-
-    # Still the reification process correctly brings back Cat since `species` is
-    # properly set to this sub-classed name.
-    expect(versions.second.reify).to be_a(Cat) # Sylvester
-    expect(versions.third.reify).to be_a(Cat) # Cheshire
-    expect(versions.fourth.reify).to be_a(Cat) # Cheshire that was destroyed
-
-    # Creating an object from the base class is correctly identified as "Animal"
-    expect(versions[5].reify).to be_an(Animal) # Muppets Drummer
-    expect(versions[6].reify).to be_an(Animal) # Animal that was destroyed
+  context "Older version entry present where item_type refers to the base_class" do
+    let(:cat) { Cat.create(name: "Garfield") }   # Index 0
+    let(:animal) { Animal.create }               # Index 4
+    let(:migrator) { ::PaperTrailSpecMigrator.new }
+
+    before do
+      # This line runs the `let` for :cat, creating two entries
+      cat.update_attributes(name: "Sylvester")   # Index 1 - second
+      cat.update_attributes(name: "Cheshire")    # Index 2 - third
+      cat.destroy                                # Index 3 - fourth
+      # With PT <= v9.2 a subclassed version's item_type referred to the base_class, but
```

Comment:
Again, 9.2 has already been released, so if you have to reference something, please just reference this PR.